hugo/assets/scss/components/media.scss

```diff
@@ -42,55 +41,21 @@
     &__caption {
         @include style-text-small;
 
-        padding: 0.5rem 0 0.25rem;
+        font-style: italic;
+        padding-right: 0;
         width: 100%;
     }
 
-    &__credits {
```

Comment:
don't we need credits and caption styling anymore?

Reply:
Caption styling is above, credits is not build-in at the moment, so that indeed does not need any styling